cmd/kedge/main.go

```diff
@@ -69,6 +69,24 @@ var (
 			"director & backendpool routings. It will update them directly into into flagz value, so you can see the current routings anytime in debug/flagz")
 )
 
+const (
+	grpcWebContentTypePrefix = "application/grpc-web"
+	grpcContentTypePrefix    = "application/grpc"
+)
+
+func isGRPCReq(header http.Header) bool {
+	// Do not treat "grpc-web" request as pure gRPC. It is pure HTTP/2 at this point.
+	if strings.HasPrefix(header.Get("content-type"), grpcWebContentTypePrefix) {
```

Comment:
`content-type` is usually written `Content-Type`. Can you check whether `header.Get` understands this?

Reply:
Yea it was like that all the time. Works fine.